content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -0,0 +1,195 @@
+edit your `dvc.yaml` file to replace `params.yaml` with `params.json`.
+
+For more about the `params.yaml` file,
+[see our docs](https://dvc.org/doc/start/experiments#defining-parameters).
+
+### [Q: Is there a guide for migrating from Git-LFS to DVC?](https://discordapp.com/channels/485586884165107732/485596304961962003/743559246599421974)
+
+We don't know of any published guide. One of our users shared their procedure
+for disabling LFS:
+
+```dvc
+$ git lfs uninstall
+$ git rm .gitattributes
+$ git rm .lfsconfig
+```
+
+Then you can `dvc add` files you wish to put in DVC tracking, and `dvc push`
+them to your remote. After that, `git commit` and you're good!
+
+Note that, if you're going to delete any LFS files, make sure you're certain the
+corresponding data has been transferred to DVC.
+
+### [Q: Is there a way to use DVC and CML to validate a model in a GitHub Action, without making the validation data available to the user opening the Pull Request?](https://discordapp.com/channels/485586884165107732/485596304961962003/739202123295883325)
+
+We don't have special support for this use case, and there may be some security
+downsides to using a confidential validation dataset with someone else's code
+(be sure nothing in their code could expose your data!). But, there are ways to
+implement this if you're sure about it.
+
+One possible approach is to create a separate "data registry" repository using a
+private cloud bucket to store your validation dataset
+([see our docs about the why and how of data registries](https://dvc.org/doc/use-cases/data-registries#data-registries)).
+Your CI system can be setup to have access to the data registry via secrets
+(called "variables" in GitLab). Then when you run validation via
+`dvc repro validate`, you could use `dvc get` to pull the private data from the
+registry.
+
+The data is never exposed to the user in an interactive setting, only on the
+runner- and there it's ephemeral, meaning it does not exist once the runner
+shuts down.
+
+## CML questions
+
+### Q: Sometimes when I make a commit on a branch, my CI workflow isn't triggered. What's going on?
+
+If your workflow is set to trigger on a push (as in the CML use cases), it isn't
+enough to `git commit` locally- you need to push to your GitHub or GitLab
+repository. If you want every commit to trigger your workflow, you'll need to
+push each one!
+
+What about if you _don't_ want a push to trigger your worfklow? In GitLab, you
+can use the
+[`[ci skip]` flag](https://docs.gitlab.com/ee/ci/yaml/#skip-pipeline)- make sure
+your commit message contains `[ci skip]` or `[skip ci]`, and GitLab CI won't run
+the pipeline in your `gitlab-ci.yml` file.
+
+In GitHub Actions, this flag isn't supported, so you can manually kill any
+workflows in the Actions dashboard. For a programmatic fix,
+[check out this workaround by Tim Heuer](https://timheuer.com/blog/skipping-ci-github-actions-workflows/).
```

Comment:
same here, looks like CI check fails bc of this link - put it into the exclusion list ?